src/main/java/seedu/address/util/Network.java

```diff
@@ -5,11 +5,26 @@
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
 import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+import org.asynchttpclient.AsyncHttpClient;
+import org.asynchttpclient.Dsl;
+import org.asynchttpclient.Response;
 
 /**
  * Manager of Network component
  */
 public abstract class Network {
+
+    private static final int CONNECTION_TIMEOUT_MILLIS = 1000 * 10; // 10 seconds
```

Comment:
Do we need to increase the timeouts to cater to users with slow Internet connections?

Reply:
You're right, we can afford to increase the timeouts a little because this operation will be done in the background.

Increased request timeout to 60 seconds